processing-service-core/api/controllers/globalConfig.js

```diff
@@ -18,21 +18,21 @@ var logger = require('../helpers/loggingHelper.js');
 var httpHelper = require('../helpers/httpPromiseHelper.js');
 
 module.exports = {
-  createGlobalConfig: createGlobalConfig,
+  setGlobalConfig: setGlobalConfig,
   deleteGlobalConfig: deleteGlobalConfig,
   getGlobalConfig: getGlobalConfig,
   getGlobalConfigs: getGlobalConfigs
 };
 
-function createGlobalConfig(req, res, next) {
+function setGlobalConfig(req, res, next) {
   var globalConfigParams = {
     key: req.swagger.params['key'].value,
-    default: req.swagger.params['default'].value,
-    description: req.swagger.params['description'].value
+    default: req.swagger.params.globalConfig.value.default,
+    description: req.swagger.params.globalConfig.value.description
   };
   logger.info('Creating global configuration using parameters: '+ JSON.stringify(globalConfigParams));
-  var createPromise = globalConfigModel.createGlobalConfig(globalConfigParams);
-  httpHelper.writeCreatePromiseJSONResultToResponse(createPromise, res);
+  var setPromise = globalConfigModel.setGlobalConfig(globalConfigParams);
+  httpHelper.writeCreatePromiseJSONResultToResponse(setPromise, res);
```

Comment:
This should be writeUpdatePromise since the contract states this only returns a 204.